### DISCUSSION TITLE
Admin Console Customization: To add new button to export users

### DISCUSSION THREAD

**Discussion by nsgeeta12** [keycloak/keycloak#23449] [Archive]
I am looking for the documentation to customize the admin console UI, Particularly the Users Section component. My requirement is to provide a button on Users options to export all the users for a selected realm.

With keycloak version <= 18, the customization was achieved with angularJs, html and javascript. With latest keycloak (or atleast with keycloak 21.1.2), I am looking for the documentation to customize the admin console UI.

Appreciate your inputs!

Thanks

**Comment by jonkoops** [comment]
See #22255, I'd recommend you weigh in the existing discussion there.
